processing/data_collection/gazette/spiders/ba_vitoria_da_conquista.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+import datetime as dt
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class BaVitoriaDaConquistaSpider(BaseGazetteSpider):
+    TERRITORY_ID = '2933307'
+    name = 'ba_vitoria_da_conquista'
+    allowed_domains = ['pmvc.ba.gov.br']
+    start_urls = ['http://dom.pmvc.ba.gov.br/']
+
+    def parse(self, response):
+        """
+        @url http://dom.pmvc.ba.gov.br/
+        @returns items 1
+        @scrapes date file_urls is_extra_edition territory_id power scraped_at
+        """
+        menu_xpath = '/html/body/div[2]/div[4]/div/div[1]'
+        menu = response.xpath(menu_xpath)
+        years_xpath = 'div[1]/ul/li/a/text()'
+        links_xpath = '//a[text()[normalize-space(.) = "Visualizar Diário"]]/@href'
+        for y in menu.xpath(years_xpath).extract():
+            for m in range(1, 13):
+                urls = response.xpath(links_xpath).extract()
+                for url in urls:
+                    yield Gazette(
+                        date=parsing_date,
```

Comment:
`parsing_date`is wrong here. BTW, this will never work, because it is an undefined variable. Did you execute this job at least once????

`response.css('.box-diario')` contains the info you need to work for each gazette.